index.html

```diff
@@ -572,6 +572,173 @@ <h3 class="sg-section-sub-title">Table Classes</h3>
                     <div class="sg-section forms-section" id="forms-example">
                         <h2 class="sg-section-title">Forms</h2>
                         
+
+    
+        <h3 class="section-sub-title">Example Form</h3>
+
+<form action="" method="post">
```

Comment:
oh yeah sorry, I should have said ignore the compiled index file, it will look silly